### ISSUE TITLE
Align with CodeMeta

### ISSUE THREAD

**Issue by stain** [ResearchObject/ro-lite#3]
We should have a [CodeMeta](https://codemeta.github.io/) dual Resarch Object example, specially as CodeMeta is now being picked up by GitHub, Zenodo, etc and it heavily uses http://schema.org/

It might be that the [Workflow example](https://researchobject.github.io/ro-lite/0.1.0/#next-step-workflow-research-object%22) can be adapted for that as arguably a workflow is also software.

Q: Should `codemeta.json` file be a valid `manifest.jsonld` alternative by adding some extra RO terms, or would both manifests live side by side? (generating one from another)

**Comment by dgarijo** [comment]
codemeta is more oriented towards software. If a workflow has different steps, shouldn't there be a codemeta.json per step? Or do you think the whole RO should have a codemeta? What if the RO does not have software?

**Comment by stain** [comment]
Exactly, so I think the codemeta files would better be linked up by annotations and conformsTo. They might be external.

However a workflow-RO could very much be seen as software, and so a codemeta.json for the workflow could very well be in scope for licensing, runtime requirements etc.

**Comment by stain** [comment]
See #275
